loculus_values/values.yaml

```diff
@@ -641,6 +644,7 @@ defaultOrganismConfig: &defaultOrganismConfig
           type: link
           url: "https://www.ncbi.nlm.nih.gov/bioproject/__value__"
         header: "INSDC"
+        includeInDownloadsByDefault: true
```

Comment:
I am not so keen on including the BioProject by default given it isn't really relevant for stuff submitted directly to PPX, and just because I'm not sure it adds enough utility.